x-pack/plugin/core/src/main/resources/logs-default-mappings.json

```diff
@@ -0,0 +1,152 @@
+{
+  "template": {
+    "mappings": {
+      "_meta": {
+      },
+      "dynamic_templates": [
+        {
+          "strings_as_keyword": {
+            "mapping": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            },
+            "match_mapping_type": "string"
+          }
+        }
+      ],
+      "date_detection": false,
+      "properties": {
+        "@timestamp": {
+          "type": "date"
+        },
+        "stream": {
+          "properties": {
+            "type": {
+              "type": "constant_keyword",
+              "value": "logs"
+            },
+            "dataset": {
+              "type": "constant_keyword"
+            },
+            "namespace": {
+              "type": "constant_keyword"
+            }
+          }
+        },
+        "agent": {
```

Comment:
I'm torn if we should add these or leave them out.

Reply:
@ruflin I would leave it. I think it ecourages good practice to have it.